modules/services/window-managers/i3-sway/sway.nix

```diff
@@ -162,6 +162,43 @@ let
         '';
       };
 
+      bindswitches = mkOption {
+        type = types.attrsOf (
+          types.attrsOf (
+            types.oneOf [
+              types.str
+              types.bool
+            ]
+          )
```

Comment:
I think this could be a submodule that accepts `reload`, `locked` and `action`

Reply:
Yes that's clearly better I agree, I changed it to use a submodule